tests/unit/conftest.py

```diff
@@ -24,18 +21,8 @@ def patch(request):
 
 
 @pytest.fixture
-def pyramid_config(pyramid_settings):
-    with testing.testConfig(settings=pyramid_settings) as config:
-        config.include("pyramid_jinja2")
-        config.include("pyramid_services")
-        add_routes(config)
-        yield config
-
-
-@pytest.fixture
-def pyramid_request(
-    pyramid_config,  # pylint:disable=unused-argument
-):
-    pyramid_request = testing.DummyRequest()
-    apply_request_extensions(pyramid_request)
-    return pyramid_request
+def pyramid_config(pyramid_config):
+    pyramid_config.include("pyramid_jinja2")
+    pyramid_config.include("pyramid_services")
+    add_routes(pyramid_config)
+    return pyramid_config
```

Comment:
Via inherits the cookiecutter's `pyramid_config` fixture and extends it.

Via can also just use the cookiecutter's `pyramid_request` fixture directly, doesn't even need to customise it